includes/Story_Post_Type.php

```diff
@@ -314,6 +315,32 @@ public static function admin_enqueue_scripts( $hook ) {
 		wp_styles()->add_data( self::WEB_STORIES_STYLE_HANDLE, 'rtl', 'replace' );
 	}
 
+	/**
+	 * Returns a list of allowed file types.
+	 *
+	 * @return array List of allowed file types.
+	 */
+	protected static function get_allowed_file_types() {
```

Comment:
Reminds me that we should add some much needed tests for this class. Opened #328 for this.